_pytest/vendored_packages/attr/__init__.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
I'm strongly opposed to any vendoring if it can be avoided. I read up on #944, which explains the motivations for vendoring pluggy. Do those same motivations apply for this package? Do those motivations not apply to any package on which pytest depends?